src/json.c

```diff
@@ -68,7 +68,7 @@ static RSLanguage SchemaRule_JsonScore(RedisModuleCtx *ctx, const SchemaRule *ru
     goto done;
   }
 
-  if(japi->getDoubleFromKey(jsonKey, rule->score_field, &score) != REDISMODULE_OK) {
+  if(japi && japi->getDoubleFromKey(jsonKey, rule->score_field, &score) != REDISMODULE_OK) {
```

Comment:
If we're not supposed to even be here unless we have `japi` set, and we cannot return an error, we might as well just assert.

Reply:
Don't we want to get out early here? `RedisJSON_GetString` is using `japi`.

Reply:
What line are you referring to?
Anyway, in `RedisJSON_GetString` there is also testing for none-null `japi`